cmd/delete/matchable_cluster_resource_attribute.go

```diff
@@ -11,20 +11,20 @@ import (
 )
 
 const (
-	clusterResourceAttributesShort = "Delete matchable resources of cluster attributes"
+	clusterResourceAttributesShort = "Deletes matchable resources of cluster attributes."
 	clusterResourceAttributesLong  = `
-Deletes cluster resource attributes for the given project and domain combination or additionally with workflow name.
+Delete cluster resource attributes for the given project and domain combination or additionally the workflow name.
 
-For project flytectldemo and development domain, it is:
+For project flytectldemo and development domain:
 ::
 
  flytectl delete cluster-resource-attribute -p flytectldemo -d development 
 
 
-Deletes cluster resource attribute using config file which was used to create it.
+Delete cluster resource attribute using the config file that was used to create it.
 Here, the config file is written to cra.yaml.
-Attributes are optional in the file as they are unread during the delete command but can be kept since the same file can be used for get, update or delete commands.
-e.g., content of cra.yaml:
+Attributes are optional in the file as they are unread during the delete command but can be retained since the same file can be used for get, update or delete commands.
```

Comment:
Can we say "They are unread during the delete command but can be retained since the same file can be used for 'get', 'update' or 'delete' commands." ?